### PR TITLE
Caveat that heroku-toolbelt does not include Foreman

### PR DESCRIPTION
**Pull request by duozmo**
Addresses https://github.com/Homebrew/homebrew/issues/40379 and http://stackoverflow.com/q/27745595/134610

### CODE REVIEW

File: Library/Formula/heroku-toolbelt.rb [hunk 1]
```diff
@@ -16,6 +16,14 @@ def install
...
   end
 
+  def caveats
+    <<-EOS.undent
+      Unlike the standalone download for Heroku Toolbelt, the Homebrew package
+      does not come with Foreman (theforeman.org). Run `gem install foreman`
+      if you would like it.
+    EOS
+  end
+
```
Comment (bfontaine) [inline Library/Formula/heroku-toolbelt.rb]: Please remove the trailing spaces here.

File: Library/Formula/heroku-toolbelt.rb [hunk 2]
```diff
@@ -16,6 +16,14 @@ def install
     bin.write_exec_script libexec/"bin/heroku"
   end
 
+  def caveats
+    <<-EOS.undent
+      Unlike the standalone download for Heroku Toolbelt, the Homebrew package
+      does not come with Foreman (theforeman.org). Run `gem install foreman`
```
Comment (mikemcquaid) [inline Library/Formula/heroku-toolbelt.rb]: This will fail with a permissions error on OS X's default Ruby, unfortunately.

**Comment by duozmo** [comment]
@mikemcquaid Whoops. I’ve been using `rbenv` so long I forgot about system Ruby.

@bfontaine Sorry, apparently I tweaked the file after `brew audit`.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/1e29417a01681fe49c46f608e88eb6bf69855e13. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Please note ideal commit message here would have been something like `heroku-toolbelt: clarify does not include foreman`. The style for non-version changes should always be `formula: something i did`, Thanks!

**Pull request closed without merging**